training_controller/prepare_training_logs.py

```diff
@@ -13,33 +13,22 @@
 from elasticsearch.helpers import scan
 
 logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(message)s")
-MINIO_SERVER_URL = os.environ["MINIO_SERVER_URL"]
-MINIO_ACCESS_KEY = os.environ["MINIO_ACCESS_KEY"]
-MINIO_SECRET_KEY = os.environ["MINIO_SECRET_KEY"]
 ES_ENDPOINT = os.environ["ES_ENDPOINT"]
 ES_USERNAME = os.environ["ES_USERNAME"]
 ES_PASSWORD = os.environ["ES_PASSWORD"]
 FORMATTED_ES_ENDPOINT = (
     f"https://{ES_USERNAME}:{ES_PASSWORD}@" + ES_ENDPOINT.split("//")[-1]
 )
+TRAINING_DATA_PATH = os.getenv("TRAINING_DATA_PATH", "/var/opni-data")
 
 
 class PrepareTrainingLogs:
-    def __init__(self, working_dir):
-        self.WORKING_DIR = working_dir
+    def __init__(self):
+        self.WORKING_DIR = TRAINING_DATA_PATH
```

Comment:
no need to have this be a global var just set to `os.getenv("TRAINING_DATA_PATH", "/var/opni-data")`